addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1672,6 +1648,23 @@ options.registry.anchorName = options.Class.extend({
         }
         this.$target.trigger('content_changed');
     },
+    /**
+     * Copies anchor to clipbboard.
+     *
+     * @private
+     */
+    _copyAnchorToClipboard: function () {
+        const $input = $("<input>");
+        $("body").append($input);
+        $input.val('#' + this.$target[0].id).select();
+        document.execCommand("copy");
+        $input.remove();
```

Comment:
I think we have a library / utility function to do this (clipboard.js ?), used to copy the traceback when you get one for example, and it will handle the copied popover too.